app/views/dashboard/index.html.erb

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang='en'>
+
+<head>
```

Comment:
it's best practice to put your links to your css and javascript in your application.html.erb so they are rendered on every view 👍